fastchat/serve/gradio_block_arena_vision_named.py

```diff
@@ -206,6 +206,20 @@ def add_text(
             * 6
         )
 
+    if image_flagged:
+        logger.info(f"image flagged. ip: {ip}. text: {text}")
+        for i in range(num_sides):
+            states[i].skip_next = True
+        return (
+            states
+            + [x.to_gradio_chatbot() for x in states]
+            + [{"text": IMAGE_MODERATION_MSG}]
+            + [
+                no_change_btn,
+            ]
+            * 6
```

Comment:
what is this *6 for?

Reply:
It corresponds to the `btn_list` variable. The idea is that if someone inputs an image that is flagged then we should not change the interface at all since they shouldn't be able to chat with it so no button changes such as buttons appearing or disabling should occur.